app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/DiffUtilCallback.java

```diff
@@ -0,0 +1,52 @@
+package com.futurice.freesound.feature.common.ui.adapter;
+
+import com.futurice.freesound.feature.common.DisplayableItem;
+
+import android.support.annotation.NonNull;
+import android.support.v7.util.DiffUtil;
+
+import java.util.List;
+
+final class DiffUtilCallback extends DiffUtil.Callback {
```

Comment:
I'm not 100% certain of how to do it, I would have thought that importing the top level settings would be enough.

If you need to import the Copyright directly then: Preferences -> Editor -> Copyright -> Copyright Profiles

You can then import the file from the repo: <repo>/ide/copyright/

And I think you need to then set that as your copyright profile for this project.